db-schema.sql

```diff
@@ -568,6 +613,8 @@ GitopsEngineInstance -> GitopsEngineCluster
 GitopsEngineCluster -> ClusterCredentials
 ManagedEnvironment -> ClusterCredentials
 
+AppProjectRepository -> ClusterUser
+AppProjectRepository -> RepositoryCredentials
```

Comment:
Shouldn't the foreign key(s) for AppProjectManagedEnvironment also be listed here?

Reply:
Yes, i will update this